task_manager_app/permissions.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import permissions
+from rest_framework.permissions import BasePermission, DjangoModelPermissions
+
+
+class IsUserAuthor(BasePermission):
```

Comment:
Хорошее разрешение. Упрощает работу в представлении. Но я бы поменял название на более семантически значимое. Например
```suggestion
class IsOwnerOrReadOnly(BasePermission):
```